capitals.rb

```diff
@@ -151,3 +152,25 @@
     name: "Wyoming",
     capital: "Cheyenne"
 }]
+
+right = 0
```

Comment:
:+1: try implementing this so that each state has its own `right` and `wrong` values so you can keep track of how well you're doing with a particular state-capital pair